toolkit/scripts/containerized-build/create_container_build.sh

```diff
@@ -0,0 +1,197 @@
+    --enable-local-repo ) enable_local_repo=true; shift ;;
+    --help ) help; exit 1 ;;
+    ? ) echo -e "ERROR: INVALID OPTION.\n\n"; help; exit 1 ;;
+  esac
+done
+
+# Assign default values
+[[ -z "${repo_path}" ]] && repo_path=$(realpath "$(dirname "$0")") && repo_path=${repo_path%'/toolkit'*}
+[[ ! -d "${repo_path}" ]] && { print_error " Directory ${repo_path} does not exist"; exit 1; }
+[[ -z "${mode}" ]] && mode="build"
+[[ -z  "${version}" ]] && version="2.0"
+
+echo "Running in ${mode} mode, requires root..."
+echo "Running as root!"
+cd "${script_dir}"  || { echo "ERROR: Could not change directory to ${script_dir}"; exit 1; }
+
+# ==================== Setup ====================
+build_dir="${script_dir}/build_container"
+mkdir -p "${build_dir}"
+
+# Get Mariner GitHub branch at $repo_path
+pushd ${repo_path}
+repo_branch=$(git rev-parse --abbrev-ref HEAD)
+popd
+
+#Set splash text based on mode
+if [[ "${mode}" == "build" ]]; then
+    splash_txt="builder_splash.txt"
+else
+    splash_txt="tester_splash.txt"
+fi
+
+# ============ Populate SRPMS ============
+# Populate ${repo_path}/build/INTERMEDIATE_SRPMS with SRPMs, that can be used to build RPMs in the container
+pushd "${repo_path}/toolkit"
+echo "Populating Intermediate SRPMs..."
+sudo make input-srpms SRPM_FILE_SIGNATURE_HANDLING="update" > /dev/null
+popd
+
+# ============ Map chroot mount ============
+if [[ "${mode}" == "build" ]]; then
+    # Create a new directory and map it to chroot directory in container
+    if [ -d "${repo_path}/build/container-build" ]; then rm -Rf ${repo_path}/build/container-build; fi
+    if [ -d "${repo_path}/build/container-buildroot" ]; then rm -Rf ${repo_path}/build/container-buildroot; fi
+    mkdir ${repo_path}/build/container-build
+    mkdir ${repo_path}/build/container-buildroot
+    mounts="${mounts} ${repo_path}/build/container-build:${topdir}/BUILD ${repo_path}/build/container-buildroot:${topdir}/BUILDROOT"
+fi
+
+# ============ Setup tools ============
+# Copy relavant build tool executables from ${repo_path}/tools/out
+echo "Setting up tools..."
+if [[ ( ! -f "${repo_path}/toolkit/out/tools/depsearch" ) || ( ! -f "${repo_path}/toolkit/out/tools/grapher" ) || ( ! -f "${repo_path}/toolkit/out/tools/specreader" ) ]]; then build_tools; fi
+if [[ ! -f "${repo_path}/build/pkg_artifacts/graph.dot" ]]; then build_graph; fi
+cp ${repo_path}/toolkit/out/tools/depsearch ${build_dir}/
+cp ${repo_path}/toolkit/out/tools/grapher ${build_dir}/
+cp ${repo_path}/toolkit/out/tools/specreader ${build_dir}/
+cp ${repo_path}/build/pkg_artifacts/graph.dot ${build_dir}/
+
+# ========= Setup mounts =========
+echo "Setting up mounts..."
+
+mounts="${mounts} ${repo_path}/out/RPMS:/mnt/RPMS"
+if [[ "${mode}" == "build" ]]; then
+    # Add extra build mounts
+    mounts="${mounts} ${repo_path}/build/INTERMEDIATE_SRPMS:/mnt/INTERMEDIATE_SRPMS ${repo_path}/SPECS:${topdir}/SPECS"
+fi
+
+rm -f ${build_dir}/mounts.txt
+for mount in $mounts $extra_mounts; do
```

Comment:
Is `extra_mounts` defined anywhere?

Reply:
Nope, missed this, thanks for catching :)